docs/TektonConfig.md

```diff
@@ -655,6 +664,14 @@ The following fields are supported in `HorizontalPodAutoscaler` (aka HPA)
 
 **NOTE**: If a Deployment or StatefulSet has a Horizontal Pod Autoscaling (HPA) and is in active state, Operator will not control the replicas to that resource. However if `status.desiredReplicas` and `spec.minReplicas` not present in HPA, operator takes the control. Also if HPA disabled, operator takes control. Even though the operator takes the control, the replicas value will be adjusted to the hpa's scaling range.
 
+#### webhookConfigurationOptions
+Defines additional options for each webhooks. Use webhook name as a key to define options for a webhook. To get detailed information about webhooks options visit https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/
```

Comment:
we should add a note for if the webhook does not exist we that name, the configuration will be ignored

Reply:
comment added